src/main/java/com/fvv/bookstore/util/DateUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.fvv.bookstore.util;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Class DateUtil to convert a String into a Date
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 30 de mar de 2018</p>	
+ * @version 1.0 	
+ *
+ */
+public final class DateUtil {
+
+	public static Date stringToDate(final String strDate) {		
+		if (strDate != null) {
+			try {
+				SimpleDateFormat format = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
put the mask in a separated constant attribute

Reply:
Ok